features/backward_compatibility.feature

```diff
@@ -0,0 +1,18 @@
+@staging-only
+Feature: Compatibility with old supported MSA versions
+  These tests check that the hub works correctly with old supported
+  MSA versions.
+
+  Scenario Outline: Sign in successful with IDP and cycle 3
+    Given the user is at "<RP>"
+    And they start a sign in journey
+    And they select IDP "Stub Idp Demo Two"
+    And they login as "stub-idp-demo-two-c3"
+    And they submit cycle 3 "AA123456A"
+    Then they should be successfully verified
+
+  Examples:
+    | RP                                                              |
+    | https://test-rp-staging-backcompat-1.cloudapps.digital/test-rp/ |
+    | https://test-rp-staging-backcompat-2.cloudapps.digital/test-rp/ |
+    | https://test-rp-staging-backcompat-3.cloudapps.digital/test-rp/ |
```

Comment:
We're going to have to remember to add new ones here, right? I guess if 4 and 5 are currently unoccupied that's _fine_

Reply:
Yeah I don't know cucumber well enough to find a way to avoid this toil :(